dnf/base.py

```diff
@@ -1612,6 +1730,21 @@ def _install_multiarch(self, query, reponame=None, strict=True):
             self._goal.install(select=sltr, optional=(not strict))
         return len(available)
 
+    def install_module(self, specs):
```

Comment:
No stict parameter here.

Reply:
It will be required if user touch any dependency problem with modules. I know the modularity is nearly perfect, but for the possibility 1 to 1000000 lets open the possibility to user to use skip-broken option.